security-analysis/security-analysis-api/src/main/java/com/powsybl/security/detectors/AbstractLimitViolationDetector.java

```diff
@@ -44,6 +49,14 @@ public void checkCurrent(Contingency contingency, Branch branch, Branch.Side sid
         checkCurrent(contingency, branch, side, branch.getTerminal(side).getI(), consumer);
     }
 
+    /**
+     * This implementation takes the current value to be checked from the Network.
+     */
```

Comment:
Defining a comment overrides the one of the inherited method. To keep track on this, you can use the `{@inherited}` tag:
```suggestion
    /**
     * {@inheritDoc}
     * <p>This implementation takes the current value to be checked from the Network.</p>
     */
```
<br/>
I saw that you copied this comment from another method of this same class. Can you also fix the 2 other methods using it?

Reply:
done